blocks/init/src/Blocks/components/modal/modal.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/**
+ * Template for the Modal Component.
+ *
+ * @package EightshiftBoilerplate.
+ */
+
+use EightshiftBoilerplateVendor\EightshiftLibs\Helpers\Components;
+
+$globalManifest = Components::getManifest(dirname(__DIR__, 2));
+$manifest = Components::getManifest(__DIR__);
+
+$modalUse = Components::checkAttr('modalUse', $attributes, $manifest);
+
+if (!$modalUse) {
+	return;
+}
+
+$componentClass = $manifest['componentClass'] ?? '';
+$additionalClass = $attributes['additionalClass'] ?? '';
+$blockClass = $attributes['blockClass'] ?? '';
+$selectorClass = $attributes['selectorClass'] ?? $componentClass;
+$componentJsClass = $manifest['componentJsClass'] ?? '';
+$componentJsToggleClass = $manifest['componentJsToggleClass'] ?? '';
+
+$modalExitButton = Components::checkAttr('modalExitButton', $attributes, $manifest);
+$modalContent = Components::checkAttr('modalContent', $attributes, $manifest);
+$modalId = Components::checkAttr('modalId', $attributes, $manifest);
+
+$modalClass = Components::classnames([
+	Components::selector($componentClass, $componentClass),
+	Components::selector($blockClass, $blockClass, $selectorClass),
+	Components::selector($additionalClass, $additionalClass),
+	Components::selector($componentJsClass, $componentJsClass),
+]);
+
+?>
+
+<div
+	class="<?php echo \esc_attr($modalClass); ?>"
+	id="<?php echo \esc_attr($modalId); ?>"
+	aria-hidden="true"
+>
+	<div
+		<?php echo \esc_attr("data-{$componentJsToggleClass}-close"); ?>
+		class="<?php echo \esc_attr("{$componentClass}__overlay"); ?>"
+		tabIndex="-1"
+	>
+		<div
+			class="<?php echo \esc_attr("{$componentClass}__dialog"); ?>"
```

Comment:
Use the `selector` helper please.

Reply:
there is no need. If you have one sector there is no need. it is ok to put it like this.